### PR TITLE
Creating class

### PR DESCRIPTION
**Pull request by HectorTorresE**
In this PR:

- Class Person added with the instance vars id, name, age. A constructor, getters, setters (Via attributes), and required methods.
- Class Student added, inherits from Person, add variable classroom and method play_hooky
- Class Teacher added, inherits from Person, add specialization variable and override can_user_services
- Each class is in a separate file

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  attr_accessor :name, age
+
+  def can_use_services?
+    @age >= 18 || parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+end
```
Comment (micheaol) [inline person.rb:21]: Please ensure that the `method is_of_age?` is a private method according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @HectorTorresE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: person.rb (hunk 1 above)
Comment (HectorTorresE) [reply person.rb:21]: Yes, the name is ``is_of_age?``, but whenever I use this name a get a linter error that asks me to rename the method to ``of_age?``

Comment (micheaol) [reply person.rb:21]: @HectorTorresE ,

You can still have the method name as `of_age?` but ensure that the method is `private`. I hope this helps.

Comment (HectorTorresE) [reply person.rb:23]: Great, thanks for the review

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @HectorTorresE , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**